README.md

```diff
@@ -95,7 +95,7 @@ cargo install --features=<name> ...
 The recommended requirements for compiling and running `zebrad` are:
 - 4+ CPU cores
 - 16+ GB RAM
-- 50GB+ available disk space for building binaries and storing finalized state
+- 100GB+ available disk space for building binaries and storing cached chain state
```

Comment:
```suggestion
- 100 GB+ available disk space for building binaries and storing cached chain state
```

Reply:
I suggest more than that, given the rate at which chain state size is increasing. Maybe 400 GB+?